js/tests/index.test.js

```diff
@@ -84,15 +88,15 @@ const run = async () => {
     setJustParseWithLang(parseLang);
     setParsePattern(parsePattern);
 
-    const testErrors = run("");
-    for (const e of testErrors) {
-      console.log(e);
-    }
+    run("");
+    process.exit(0);
   } catch (e) {
     console.log("Error running tests:\n");
     console.log(e);
+    process.exit(1);
   } finally {
     console.log("Finished running tests");
```

Comment:
hmm this finally will never execute then because of the process.exit(0) above.

Reply:
Ah yep, was moving too fast. I'm not even sure if we need the logging anymore -- I think that was more of an Austin debugging thing